packages/components/src/store/vsCode.js

```diff
@@ -67,6 +75,13 @@ export function buildStore() {
       selectedSavedFilter: null,
       defaultFilter: null,
       highlightedEvents: [],
+
+      limitRoot: false,
+      hideMedia: false,
+      hideExternalCode: false,
+      hideUnlabeled: false,
+      hideElapsedTimeUnder: false,
+      hideName: false,
```

Comment:
Instead of creating these new state variables in the store, take a look at the `state.filters.declutter` property to keep track of the filter state.

Reply:
Thanks for your review. I got what you mean and I will update the code asap.